contrib/README.md

```diff
@@ -0,0 +1,24 @@
+contrib
+=======
+
+## Overview
+
+This consists of extra optional tools which may be useful when working with
+dcrd and related software.
+
+## Contents
+
+### Example Service Configurations
+
+- [rc.d](services/rc.d/dcrd)  
+  Provides an example `rc.d` script for configuring dcrd as a background service
```

Comment:
should mention that this is specifically for OpenBSD.  Can't say it will run on, say, FreeBSD.

Reply:
Good point.  I updated this to say:

```
  Provides an example `rc.d` script for configuring dcrd as a background service
  on OpenBSD.  It also serves as a good starting point for other operating
  systems that use the rc.d system for service management.
```